optimize/self-managed/optimize-deployment/migration-update/2.2-to-2.3.md

```diff
@@ -52,4 +52,4 @@ Apart from caching issues, there is the following list of known data update limi
 
 The configuration of Optimize allows you to define when the history cleanup is triggered using cron expression notation. However, the values are incorrectly interpreted in Optimize. For example, the `historyCleanup.cronTrigger` configuration has the default value `0 1 * * *`, which should be 01:00 AM every day. Unfortunately, a bug causes this to be interpreted as every hour.
 
-To fix this, use the Spring [cron expression notation](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/scheduling/support/CronSequenceGenerator.html). For instance, the default value for `historyCleanup.cronTrigger` would then be `0 0 1 * * *`.
+To fix this, use the Spring [cron expression notation](https://docs.spring.io/spring-framework/docs/5.1.0.RELEASE/javadoc-api/org/springframework/scheduling/support/CronSequenceGenerator.html). For instance, the default value for `historyCleanup.cronTrigger` would then be `0 0 1 * * *`.
```

Comment:
I am not going back to check which version of Spring we used on version 2.3... 😆 

I would consider removing it here and in all the files below though, I don't see a big value including a link here

Reply:
There are 30 references of this across the docs -- would you prefer to remove the entire `### Misinterpreted cron expressions` section? If so, do you feel users really need this context often, or it is rarely used?

If not, would you prefer to link to something like https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/support/CronExpression.html?

Reply:
I don't have a good judgement on how often this is used, but I am sure it was probably added in the first place because it would have confused users at some point in time. I would be happy to change it to the docs link you suggested. This is the correct cron expression type and is not versioned docs. Thanks for the idea!